README.rst

```diff
@@ -25,28 +25,68 @@ requests directly to the Civis API. See the
 `full documentation <https://civis-python.readthedocs.io>`_ for more details.
 
 
-Installation
-------------
+API Keys
+--------
+
+Usage of ``civis-python`` requires a valid Civis Platform API key, which can
+be created by following the instructions
+`here <https://civis.zendesk.com/hc/en-us/articles/216341583-Generating-an-API-Key>`_.
+API keys have a set expiration date and new keys will need to be created at
+least every 30 days. ``civis-python`` will look for a ``CIVIS_API_KEY``
+environmental variable to access your API key, so after creating a new API key,
+follow the steps below for your operating system to set up your environment.
 
-1. Get a Civis API key `(instructions) <https://civis.zendesk.com/hc/en-us/articles/216341583-Generating-an-API-Key>`_.
-2. Add a ``CIVIS_API_KEY`` environment variable.
-3. You can add the following to ``.bash_profile`` for bash::
+Linux / MacOS
+~~~~~~~~~~~~~
+
+1. Add the following to ``.bash_profile`` for bash::
 
     export CIVIS_API_KEY="alphaNumericApiK3y"
 
-4. Source your ``.bash_profile``
-5. Install the package::
+2. Source your ``.bash_profile`` (or restart your terminal).
+
+Windows 10
```

Comment:
It might be worth mentioning https://docs.microsoft.com/en-us/windows/wsl/install-win10, which in my opinion is the best way to use python in windows.

Reply:
I suspect users will already have a workflow for python on Windows before they install `civis`. This is pretty cool, though.